### PR TITLE
cargo-deny: update to 0.10.0.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by ericonr** [comment]
Oh nice, you've opened an issue already !

File: srcpkgs/cargo-deny/template [hunk 1]
```diff
@@ -11,12 +11,17 @@ maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
...
 
 if [ "$XBPS_TARGET_ENDIAN" = "be" ]; then
 	broken="bitvec crate unimplemented for big endian"
 fi
 
+case "$XBPS_TARGET_MACHINE" in
+	# See https://github.com/EmbarkStudios/cargo-deny/issues/372
+	i686*) make_checK-args="-- --skip advisories::helpers::test::converts_url_to_path --skip index::bare::test::matches_cargo";;
+esac
+
```
Comment (ericonr) [inline srcpkgs/cargo-deny/template:24]: Typo in `K` :p 

And put it above the `broken` conditional

**Comment by ericonr** [comment]
That's probably broken on all 32bit platforms, not only i686.

**Comment by jcgruenhage** [comment]
upstream fixed it and made a new release with that fix, so here we go.

**Pull request merged**